{user} assignment

mujiindriani opened the pull request.

mujiindriani committed:
condition

mujiindriani committed:
change

mujiindriani committed:
solving-EI

ikhsan017 commented on js/beam-analysis.js:
```diff
@@ -102,45 +102,43 @@ BeamAnalysis.analyzer = {};
-    var E = beam.material.properties.E;
-    var I = beam.material.properties.I;
-
+    var EI = beam.material.properties.EI;
```
Malam mbak @mujiindriani 

sepertinya hasil perhitunganya belum sesuai ekspektasi ya, untuk simply supported, ekspektasi graph nya seperti yang di gambar yang ada di file `readme.md`

untuk 2 span unequal, ekspektasinya seperti tertera

<img width="845" alt="Screenshot 2024-03-10 at 10 01 10" src="https://github.com/mujiindriani/beamAnalysis/assets/1480166/5af7a014-9bd6-4458-af8d-efb597b8b250">

<img width="924" alt="Screenshot 2024-03-11 at 19 12 03" src="https://github.com/mujiindriani/beamAnalysis/assets/1480166/6aaceb2e-2684-482b-b711-c13579df221e">

<img width="887" alt="Screenshot 2024-03-11 at 19 13 07" src="https://github.com/mujiindriani/beamAnalysis/assets/1480166/74ecf26f-6cf5-4ae1-a482-f9f40a7f0e6a">

mujiindriani replied:
Malam pak @ikhsan017,  baik pak saya coba review kembali untuk rumusnya dan menyesuaikan kembali  untuk perhitungan graph

mujiindriani committed:
Update beam-analysis.js

Untuk perhitungan Simply supported sudah sesuai dengan expektasi gambar, namun untuk Two span unequal output grafik masih menampilkan rentang WL1
